src/less/components/search/SearchFacade.less

```diff
@@ -0,0 +1,42 @@
+/*
+  Mixins to style <hx-search> facade states.
+*/
+
+.SearchFacade(pristine) {
```

Comment:
Remove `pristine` variant, it's never used, and rarely needed

Reply:
Block removed.